app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -685,6 +718,12 @@ class MultipleDisposalsTriageController @Inject() (
                       examplePropertyDetailsAnswers = None,
                       yearToDateLiabilityAnswers = None,
                       supportingEvidenceAnswers = None
+                    ),
```

Comment:
if the user had selected indirect only and has now not selected indirect only you'll need to convert from `DraftMultipleIndirectDisposalsDraftReturn` to `DraftMultipleDisposalsDraftReturn` and vice versa.